app/projects/services.py

```diff
@@ -295,28 +295,60 @@ def get_pv_output(self, coordinates):
             "capacity": 1.0,
             "system_loss": 0.1,
             "tracking": 0,
-            "tilt": 35,
+            "tilt": coordinates["lat"],
             "azim": 180,
             "format": "json",
+            "raw": "true",
         }
 
         r = self.s.get(url, params=args)
+        logger.info("Sending request to renewables.ninja")
+        data = self.fetch_and_parse_data(r)
+        self.data["pv"] = data
+        return
 
-        # Parse JSON to get a pandas.DataFrame of data and dict of metadata
-        parsed_response = json.loads(r.text)
+    def get_wind_data(self, coordinates):
+        ##
+        # Get Wind data
+        ##
 
-        pv_data = pd.read_json(json.dumps(parsed_response["data"]), orient="index")
-        metadata = parsed_response["metadata"]
+        url = self.api_base + "data/wind"
 
-        self.data = pv_data
+        args = {
+            "lat": 34.125,
+            "lon": 39.814,
```

Comment:
This is hardcoded for wind?

Reply:
Ah, good catch. I copy/pasted from RN and forgot to update this.